src/transform3d.rs

```diff
@@ -177,6 +177,18 @@ where T: Copy + Clone +
         )
     }
 
+    /// Check whether shapes on the XY plane with Z pointing towards the
+    /// screen transformed by this matrix would be facing back.
```

Comment:
I think the comment is right. It says "XY plane with Z pointing towards the screen" transformed by "this matrix" would be facing back. Which means a plane facing the screen become facing back after transformed by this matrix.